accepted/010-database.md

```diff
@@ -0,0 +1,41 @@
+# SQLite for Auraed
+
+This feature will accomplish **data persistency after reboots/restarts/failure** because **auraed is a systems daemon built for humans who often make mistakes**.
+
+### Outcomes
+
+  - Allow **persistent records** to exist after a daemon crash, failure.
+  - Allow **persistent records** to exist atomicaly after a gRPC request return. 
+      - There will be a deliberate coupling between the gRPC server and the data layer/model.
+  - Allow **encryption at rest** which can be unlocked using the `/etc/aurae/pki/server.key` (or a cryptographic hash) server key for the database.
+  - Allow **joining** and **connecting** records and schemas. For example we should have a collection of `VirtualMachine` records, which need to be mapped to `NIC` records.
+  - Allow **for a single database file** which can easily be backed up, recovered, and snapshotted for disaster recovery.
+  - Alow **embedding** the database directly into the `auraed` daemon such that another server/client relationship does not need to be maintained, or such that auraed will scheduele the database directly.
+
+### Goals
+
+ - All **auraed** daemons will ship with this **database**.
```

Comment:
does this line mean the file is the same across nodes even if they dont know that? if so, what guarantees it _is_ the same file?

Reply:
this just means that the database file is not configurable 

a team can't start the daemon in "dev mode" and have it point to this database over here... and then restart it in "stage mode" and have it point to this database over there